src/Bicep.LangServer/Snippets/Templates/res-aks-cluster.bicep

```diff
@@ -0,0 +1,30 @@
+﻿resource aksCluster1 'Microsoft.ContainerService/managedClusters@2020-02-01' = {

```

Comment:
We should start a place to consider changes to make the snippets more consistent.  E.g.:
1) Don't add anything that's not necessary (e.g. empty tags, properties objects)
2) Either be consistent with displayName or remove them all

Reply:
I removed empty tags and added displayName everywhere as part of below commit. Thanks!
https://github.com/Azure/bicep/commit/e74b8da5cc561cc934a1b5c21335071ba3594efd

Reply:
Not important for this PR, but whether we add displayName should be discussed with Brian, Alex and Marcus.

Reply:
Per offline conversation, removed the displayName tag